mmpy_bot/mattermost.py

```diff
@@ -167,26 +167,28 @@ def user(self, user_id):
 
 
 class MattermostClient(object):
-    def __init__(self, url, team, email, password, ssl_verify=True, login=1):
+    def __init__(self, url, team, email, password, ssl_verify=True, token=None):
         self.users = {}
         self.channels = {}
         self.mentions = {}
-        self.api = MattermostAPI(url, ssl_verify)
+        self.api = MattermostAPI(url, ssl_verify, token)
         self.user = None
-        self.info = None
+        # self.info = None
```

Comment:
If you are getting rid of info, might as well delete these lines instead of commenting them out.